### DISCUSSION TITLE
Dúvida: PHP ou JavaScript?

### DISCUSSION THREAD

**Discussion by yellowreach** [frontendbr/forum#1908] [Discussão]
Montei um plano de estudos, assim que eu acabasse os cursos de PHP, eu iria para os de Javascript.

Pesquisei algumas vagas voltadas para Front End no RJ, para saber como anda o mercado de trabalho, encontrei muitas vagas para Javascript, agora estou em um dilema, foco nas 2 linguagens, ou foco na que tem mais vagas? Quase não encontro vagas de PHP e quando encontro pedem conhecimento de Javascript também, estou enlouquecendo kkkk Alguém me dê um norte!

editar

**Comment by expalmer** [comment]
Estude Javascript.

**Comment by ninetails** [comment]
O básico vc já sabe... Só trocar os `->` por `.` e tirar os `$`

Brincadeiras a parte, ambas as linguagens são bem parecidas, poucas são suas particularidades... a maior diferença tá no caso de pedirem Node, que requer que vc escreva um código mais "não blocante", mas no mais... geralmente quando pedem JS para vagas PHP, jQuery já ajuda bastante. O resto é só Googlar.

Se ir além disso, aí acaba competindo a front-end (quando chega a parte de frameworks no geral), a não ser que o empregador queira unicórnios fullstacks.

OU

"eooo aiooo WOLOLOOO" e vire front-end tb ;D

**Comment by yellowreach** [comment]
> O básico vc já sabe... Só trocar os `->` por `.` e tirar os `$`
> 
> Brincadeiras a parte, ambas as linguagens são bem parecidas, poucas são suas particularidades... a maior diferença tá no caso de pedirem Node, que requer que vc escreva um código mais "não blocante", mas no mais... geralmente quando pedem JS para vagas PHP, jQuery já ajuda bastante. O resto é só Googlar.
> 
> Se ir além disso, aí acaba competindo a front-end (quando chega a parte de frameworks no geral), a não ser que o empregador queira unicórnios fullstacks.
> 
> OU
> 
> "eooo aiooo WOLOLOOO" e vire front-end tb ;D

KKKKKKKK, ajudou muito!!!!!!

Obrigado a todos ajudaram muito!
